src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/payment/FeesAndStatusCheckPreparer.java

```diff
@@ -96,65 +97,58 @@ public PreSubmitCallbackResponse<AsylumCase> handle(
         final PaymentStatus paymentStatus = asylumCase.read(PAYMENT_STATUS, PaymentStatus.class)
             .orElse(PaymentStatus.PAYMENT_PENDING);
 
-        final String paymentOptionNotAvailableLabel = "The Make a payment option is not available.";
-
-        final String oldOrExistingCasesLabel = "You cannot make a payment for this appeal using Payment by Account";
-
         YesOrNo isRemissionsEnabled
             = featureToggler.getValue("remissions-feature", false) ? YesOrNo.YES : YesOrNo.NO;
         asylumCase.write(IS_REMISSIONS_ENABLED, isRemissionsEnabled);
 
+        Optional<RemissionDecision> remissionDecision = asylumCase.read(REMISSION_DECISION, RemissionDecision.class);
+
+        boolean remissionPartiallyApproved = remissionDecision.isPresent()
+                                             && remissionDecision.get().equals(RemissionDecision.PARTIALLY_APPROVED);
+
+        boolean remissionApproved = remissionDecision.isPresent()
+                                    && remissionDecision.get().equals(RemissionDecision.APPROVED);
+
+        boolean paymentAppealEvent = callback.getEvent().equals(PAYMENT_APPEAL);
+        boolean isPaid = paymentStatus.equals(PAID);
+
         asylumCase.read(APPEAL_TYPE, AppealType.class)
             .ifPresent(type -> {
-                Optional<RemissionDecision> remissionDecision = asylumCase.read(REMISSION_DECISION, RemissionDecision.class);
                 switch (type) {
                     case EA:
                     case HU:
                     case EU:
-                        Optional<String> eaHuPaymentType = asylumCase.read(EA_HU_APPEAL_TYPE_PAYMENT_OPTION, String.class);
-                        if (eaHuPaymentType.isEmpty()
-                            && (remissionDecision.isEmpty() || remissionDecision.get() != RemissionDecision.REJECTED)
-                            && (callback.getEvent() == Event.PAYMENT_APPEAL)
-                        ) {
-                            asylumCasePreSubmitCallbackResponse.addError(oldOrExistingCasesLabel);
-
+                        if (paymentStatus.equals(PAID)
+                            || remissionPartiallyApproved
+                            || remissionApproved) {
+                            asylumCasePreSubmitCallbackResponse.addError(PAYMENT_OPTION_NOT_AVAILABLE_LABEL);
```

Comment:
Changed obsolete condition with a new one to prevent payment of paid cases or cases with full/partial remissions